distributed/tensor_parallelism/fsdp_tp_example.py

```diff
@@ -46,6 +47,12 @@
 https://docs.google.com/presentation/d/17g6WqrO00rP3MsxbRENsPpjrlSkwiA_QB4r93_eB5is/
 """
 
+_min_gpu_count = 2
```

Comment:
I think the `min_gpu_count` for FSDP + TP would be 4? at least we need 2 x 2 mesh for 2D parallel

Reply:
great catch...was thinking of just tp when I added that. 
will update.